code/game/objects/items/rcd/RTD.dm

```diff
@@ -289,8 +289,12 @@
 	if(!checkResource(selected_design.cost, user))
 		qdel(rcd_effect)
 		return ITEM_INTERACT_BLOCKING
-	var/beam = user.Beam(floor, icon_state = "light_beam", time = delay)
-	playsound(loc, 'sound/effects/light_flicker.ogg', 50, FALSE)
+	var/beam
+	if(ranged)
```

Comment:
What is ranged var supposed to do exactly? RTD by default is a ranged weapon and your change makes normal RTD's not have a beam which is unintentional I assume?

Reply:
Also I see no reason to remove the beam effect from RTD for cybor, just remove this var entirely

Reply:
Plus this is incorrectly coded as it makes normal RTD'S loose their beam effect

Reply:
same reason why janitor doesnt have bluespace light replacer and engineer borg doesnt have bluespace RPED. reminder - pr intended to replace broken tile placer (which was 1 tile range aswell)

Reply:
Please rename the var to something other than `ranged`, RTD by default is a ranged weapon so it's confusing to set it to false by default

Reply:
Please rename this var to something other than `ranged`. RTD by default is a ranged weapon so its counter intuitive to set it to FALSE by default